src/com/sap/piper/analytics/InfluxData.groovy

```diff
@@ -38,4 +38,41 @@ class InfluxData implements Serializable{
     public static void reset(){
         instance = null
     }
+
+    public static void readFromDisk(script) {
+        script.echo "Transfer Influx data"
+        def pathPrefix = '.pipeline/influx/'
+        List influxDataFiles = script.findFiles(glob: "${pathPrefix}**")?.toList()
+
+        influxDataFiles.each({f ->
+            script.echo "Reading file form disk: ${f}"
+            List parts = f.toString().replace(pathPrefix, '')?.split('/')?.toList()
+
+            if(parts?.size() == 3){
+                def type = parts?.get(1)
+
+                if(type in ['fields', 'tags']){
+                    def value = script.readFile(f.getPath())
+                    /*
+                    // handle boolean values
```

Comment:
@OliverNocon looks like this results in influx not accepting (quietly) this measurement. Any idea why?

Reply:
From the Jenkins Logs:

```
Could not report to InfluxDB. Ignoring Exception.
org.influxdb.InfluxDBException$FieldTypeConflictException: partial write: field type conflict: input field "sonar" on measurement "step_data" is type boolean, already exists as type string dropped=1
	at org.influxdb.InfluxDBException.buildExceptionFromErrorMessage(InfluxDBException.java:144)
	at org.influxdb.InfluxDBException.buildExceptionForErrorState(InfluxDBException.java:173)
	at org.influxdb.impl.InfluxDBImpl.execute(InfluxDBImpl.java:827)
	at org.influxdb.impl.InfluxDBImpl.write(InfluxDBImpl.java:460)
	at jenkinsci.plugins.influxdb.InfluxDbPublicationService.writeToInflux(InfluxDbPublicationService.java:343)
	at jenkinsci.plugins.influxdb.InfluxDbPublicationService.perform(InfluxDbPublicationService.java:298)
	at jenkinsci.plugins.influxdb.InfluxDbPublisher.perform(InfluxDbPublisher.java:222)
	at jenkinsci.plugins.influxdb.InfluxDbStepExecution.run(InfluxDbStepExecution.java:37)
	at jenkinsci.plugins.influxdb.InfluxDbStepExecution.run(InfluxDbStepExecution.java:11)
	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

Reply:
⚠️ I had to drop the whole measurement to fix the type issue. Before activating this for other steps we should be sure about the type to not break anything on DB side.